tests/integration/pure_cairo_vm/contracts/roll_warp_tester.cairo

```diff
@@ -3,49 +3,71 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin
```

Comment:
Don't you think we should split this into 2 files, one for `roll` and one for `warp`?